spec/features/peeps_spec.rb

```diff
@@ -0,0 +1,12 @@
+feature 'message' do
+  scenario 'allow to post a peep' do
+    visit('/peep')
+    # click_button('Submit')
+    fill_in 'peep', :with => 'Hi, how are you?'
+    click_button('Send')
+
+    connection = PG.connect(dbname: 'chitter_test')
```

Comment:
can you explain this bit to me? is the insert part not covered by another method?